paimon-hive/paimon-hive-connector-common/src/main/java/org/apache/paimon/hive/HiveSchema.java

```diff
@@ -200,9 +200,15 @@ private static Optional<TableSchema> getExistingSchema(
         Path path = new Path(location);
         Options options = HiveUtils.extractCatalogConfig(configuration);
         options.set(CoreOptions.PATH, location);
+        if (!options.contains(CoreOptions.BRANCH)) {
+            options.set(CoreOptions.BRANCH, "");
+        }
```

Comment:
NIT: Don't set default value here. Use different constructor of `SchemaManager` below.